Makefile

```
@@ -647,7 +648,11 @@ install-manpages: ## Install manpages the Cilium CLI.
 	cp man/* /usr/local/share/man/man1/
 	mandb
 
+api-check:
```

Should this have a doc string to show up in `make help`?

```suggestion
api-check: ## Check generated API for mutation permissions checks
```